### ISSUE TITLE
how to inference a result?

### ISSUE THREAD

**Issue by Fengww2014** [SafeAILab/zkDL#4]
thanks for your works, but i want to know how to inference a result, for example, how to classify a mnist picture.

**Comment by jvhs0706** [comment]
Hello, I do not fully understand what you mean by saying "inference a result" in general. For the classification of mnist picture, the verifiable inference is from the input to the first layer to the output of the last layer (the logits). If you are referring to selecting the class with the highest probability (i.e. largest logit), that is not covered as in most of the previous works.
________________________________
From: Fengww2014 ***@***.***>
Sent: November 6, 2024 02:08
To: SafeAILab/zkDL ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [SafeAILab/zkDL] how to inference a result? (Issue #4)


thanks for your works, but i want to know how to inference a result, for example, how to classify a mnist picture.
